### PR TITLE
US #55: Custom fields

### PR DESCRIPTION
**Pull request by bameda**
Depends on https://github.com/taigaio/taiga-front/pull/361

### CODE REVIEW

File: back/taiga_contrib_hall/templates/taiga_contrib_hall/field-diff.jinja [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
     {% if values.changed %}
         {% for att in values['changed'] %}
-            <b>{{ _("Updated attachment") }}:</b> <a href="{{ att.url }}">{{ att.filename }}</a> {% if att.changes.is_deprecated %} {% if att.changes.is_deprecated.1 %}_deprecated_{% else %}_not deprecated_{% endif %}{% endif %}{% if att.changes.description %}{{ att.changes.description.1 }}{% endif %}
+            <b>{{ _("Updated attachment") }}:</b> <a href="{{ att.url }}">{{ att.filename }}</a> {% if att.changes.is_deprecated %} {% if att.changes.is_deprecated.1 %}deprecated{% else %}not deprecated{% endif %}{% endif %}{% if att.changes.description %}{{ att.changes.description.1 }}{% endif %}
```
Comment (jespino) [inline back/taiga_contrib_hall/templates/taiga_contrib_hall/field-diff.jinja]: This change isn't related to custom fields

Comment (bameda) [inline back/taiga_contrib_hall/templates/taiga_contrib_hall/field-diff.jinja]: image

**Comment by jespino** [comment]
Not working :( don't show custom fields data in hall. Show the from, and the to, but not the values.

**Comment by bameda** [comment]
It's working for me. Are you sure you test this brancha?

**Comment by jespino** [comment]
Works, was an error on my tests.

**Pull request merged**